### PR TITLE
Grammar hacks 3.0

### PR DESCRIPTION
**Pull request by favonia**
My shining magical orb. ✨🔮✨

### CODE REVIEW

**Comment by favonia** [comment]
One more character freed.

**Comment by favonia** [comment]
@cangiuli I am not sure if I removed the highlighting of `@` correctly.

File: vim/syntax/cooltt.vim [hunk 1]
```diff
@@ -27,7 +27,7 @@ syn keyword coolttKeyw cof sub path coe hcom com hfill
 
 syn keyword coolttDecl def let normalize quit
 
-syn match   coolttSymb '=>\|[|,*×:=λ_@𝕀𝔽∂∧∨→]\|->\|#t\|#f'
+syn match   coolttSymb '=>\|[|,*×:=λ_𝕀𝔽∂∧∨→]\|->\|#t\|#f'
```
Comment (jonsterling) [inline vim/syntax/cooltt.vim:30]: Let's remove that lambda from here, it's no longer a keyword.

**Comment by favonia** [comment]
BTW no one is using type annotations now :crying_cat_face:

**Comment by jonsterling** [comment]
> BTW no one is using type annotations now 😿

They are used in most let bindings.

**Comment by favonia** [comment]
> > BTW no one is using type annotations now crying_cat_face
> 
> They are used in most let bindings.

That is handled by a different part of the grammar. BTW, my ✨🔮✨ is so shinning that even that part is improved.

**Pull request merged**